frontend/src/components/App.js

```diff
@@ -6,7 +6,10 @@ import Farm from "./Farm";
 const App = () => (
   <DataProvider
     endpoint="api/farm"
-    render={data => <Farm farm={data[0]}/>}
+    render={data => (
+      <div className="wrapper--container">
```

Comment:
This class name isn't used

Reply:
Ah, I believe it's just `wrapper` now. My bad!